README.md

```diff
@@ -1 +1,17 @@
 # Tic-Tac-Toe
+
+[![Build status](https://travis-ci.org/palledorous/Tic-Tac-Toe.svg?master)](https://travis-ci.org/palledorous)
+
+## Usage
+
+1. To start the game, run `<insert command here>`
```

Comment:
Let's figure out what you want the entrypoint for the game to be, and add it here.